tests/context/test_case_context.py

```diff
@@ -0,0 +1,176 @@
+from response_operations_ui.contexts.case import build_response_status_context
+
+
+def test_case_context_with_permissions(
```

Comment:
The test name needs a tweak, we are testing the response status

Reply:
Updated